### DISCUSSION TITLE
Does library support sanitization of include statement and any links to .css and .js file?

### DISCUSSION THREAD

**Discussion by FaizPrasla** [mganss/HtmlSanitizer#425] [General]
Does library support sanitization of include statement and any links to .css and .js file

**Comment by mganss** [comment]
I'm not sure I understand. Do you have an example?

**Reply by FaizPrasla** [reply]
"#include "fileName" and any links that can potentially open up a file

**Reply by mganss** [reply]
`#include` looks like a C preprocessor directive. Do you mean `@import` CSS rules? These are not automatically resolved or do you mean something else? If so, could you show a complete example?

**Reply by FaizPrasla** [reply]
Thank you so much. I get what you are saying. Found a different solution for it.

**Reply by FaizPrasla** [reply]
Resolved
